### PR TITLE
OU-259: Remove `public~` i18n translation references

### PR DESCRIPTION
**Pull request by zhuje**
Update translations to only reference plugin__monitoring-plugin.json and not public.json from the openshift/console.

Related to this issue: https://github.com/openshift/console/pull/14192#issuecomment-2317450254

1. References to `~public` were removed. 
2. Ran `yarn i18n` to migrate the translations into plugin__monitoring-plugin.json 
3. Updated configuration files used to generate `en > plugin__monitoring-plugin.json` by mimicking files found in https://github.com/openshift/troubleshooting-panel-console-plugin/tree/main/web/i18n-scripts

### CODE REVIEW

File: web/src/components/console/utils/status-box.tsx [hunk 1]
```diff
@@ -20,26 +20,24 @@ const LoadError: React.FC<LoadErrorProps> = ({ label, className, message, canRet
...
-              type="button"
-              onClick={window.location.reload.bind(window.location)}
-              variant="link"
-              isInline
-            >
-              try again
-            </Button>
-            .
-          </Trans>
+          {t('Please ')}{' '}
```
Comment (jgbernalp) [inline web/src/components/console/utils/status-box.tsx:31]:
```suggestion
          {t('Please')}{' '}
```

File: web/src/components/console/utils/status-box.tsx [hunk 2]
```diff
@@ -75,7 +73,7 @@ export const EmptyBox: React.FC<EmptyBoxProps> = ({ label }) => {
   return (
     <Box>
       <div data-test="empty-message" className="pf-u-text-align-center">
-        {label ? t('public~No {{label}} found', { label }) : t('public~Not found')}
+        {label ? t('~No {{label}} found', { label }) : t('Not found')}
```
Comment (jgbernalp) [inline web/src/components/console/utils/status-box.tsx:76]:
```suggestion
        {label ? t('No {{label}} found', { label }) : t('Not found')}
```

File: web/src/components/console/utils/status-box.tsx (hunk 1 above)
Comment (zhuje) [reply web/src/components/console/utils/status-box.tsx]: Applied the suggested changes in the latest commit.

File: web/src/components/console/utils/status-box.tsx (hunk 2 above)
Comment (zhuje) [reply web/src/components/console/utils/status-box.tsx]: Applied the suggested changes in the latest commit.

File: web/locales/en/plugin__monitoring-plugin.json [hunk 3]
```diff
@@ -102,14 +115,14 @@
...
-  "Last {{count}} week_one": "Last {{count}} week",
-  "Last {{count}} week_other": "Last {{count}} weeks",
+  "Last {{count}} minute_one": "Last {{count}} minute_one",
+  "Last {{count}} minute_other": "Last {{count}} minute_other",
+  "Last {{count}} hour_one": "Last {{count}} hour_one",
+  "Last {{count}} hour_other": "Last {{count}} hour_other",
+  "Last {{count}} day_one": "Last {{count}} day_one",
+  "Last {{count}} day_other": "Last {{count}} day_other",
+  "Last {{count}} week_one": "Last {{count}} week_one",
+  "Last {{count}} week_other": "Last {{count}} week_other",
```
Comment (jgbernalp) [inline web/locales/en/plugin__monitoring-plugin.json:125]: We need to keep the correct translations here

File: web/locales/en/plugin__monitoring-plugin.json [hunk 4]
```diff
@@ -144,14 +156,14 @@
...
-  "{{count}} day_one": "{{count}} day",
-  "{{count}} day_other": "{{count}} days",
+  "{{count}} second_one": "{{count}} second_one",
+  "{{count}} second_other": "{{count}} second_other",
+  "{{count}} minute_one": "{{count}} minute_one",
+  "{{count}} minute_other": "{{count}} minute_other",
+  "{{count}} hour_one": "{{count}} hour_one",
+  "{{count}} hour_other": "{{count}} hour_other",
+  "{{count}} day_one": "{{count}} day_one",
+  "{{count}} day_other": "{{count}} day_other",
```
Comment (jgbernalp) [inline web/locales/en/plugin__monitoring-plugin.json:166]: same here

File: web/src/components/console/graphs/graph-empty.tsx [hunk 5]
```diff
@@ -20,7 +20,7 @@ export const GraphEmpty: React.FC<GraphEmptyProps> = ({ height = 180, loading =
         <div className="skeleton-chart" data-test="skeleton-chart" />
       ) : (
         <div className="text-secondary" data-test="datapoints-msg">
-          {t('public~No datapoints found.')}
+          {t('No datapoints found.')}
```
Comment (jgbernalp) [inline web/src/components/console/graphs/graph-empty.tsx:23]: if we use it here, we need to initialize the translation namespace in the `useTranslation` hook

File: web/locales/en/plugin__monitoring-plugin.json (hunk 3 above)
Comment (zhuje) [reply web/locales/en/plugin__monitoring-plugin.json]: I updated this in the latest commit.

File: web/src/components/console/graphs/graph-empty.tsx (hunk 5 above)
Comment (zhuje) [reply web/src/components/console/graphs/graph-empty.tsx:23]: I updated this in the latest commit.

File: web/locales/en/plugin__monitoring-plugin.json (hunk 4 above)
Comment (zhuje) [reply web/locales/en/plugin__monitoring-plugin.json]: I updated this in the latest commit.

File: web/src/components/console/utils/status-box.tsx [hunk 6]
```diff
@@ -20,26 +20,24 @@ const LoadError: React.FC<LoadErrorProps> = ({ label, className, message, canRet
     <Box className={className}>
       <div className="pf-u-text-align-center cos-error-title">
         {_.isString(message)
-          ? t('public~Error Loading {{label}}: {{message}}', {
+          ? t('Error Loading {{label}}: {{message}}', {
```
Comment (jgbernalp) [inline web/src/components/console/utils/status-box.tsx:23]: We need o update all occurrences of `useTranslation` to add `'plugin__monitoring-plugin'`

**Pull request merged**